docs/manage/data/permissions.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Data Management Permissions"
+linkTitle: "Permissions"
+description: "Data management permissions."
+weight: 50
+type: "docs"
+tags: ["data management", "cloud"]
+# SME: Devin Hilly
+---
+
+When you view the [**DATA** tab](https://app.viam.com/data/view) while authenticated to an {{< glossary_tooltip term_id="organization" text="organization" >}}, you see data from all resources you are an owner of within that organization.
+This includes data from any resources of locations shared with organizations you own.
```

Comment:
Maybe just "This includes data from shared locations." or "This includes data from locations shared with the organization."

Reply:
I like these suggestions; implementing.